fuel-indexer/indexer/src/bin/service/main.rs

```diff
@@ -0,0 +1,86 @@
+use anyhow::Result;
```

Comment:
I think it'd be cleaner to just call this whole file `service.rs` and eliminate the container directory. Then we also wouldn't need the custom `[[bin]]` targets in the cargo.toml.